docs/ref-datastore-getting-started.md

```diff
@@ -101,3 +101,36 @@ The `DataStore` constructor takes a schema version parameter(defaults to 1).
 ```typescript
 const dataStore = new DataStore(DB_NAME, 2);
 ```
+
+## Using the CLI tool
+
+We provide a graphback plugin to generate required config for datastore.
+Read about graphback-cli [here](https://graphback.dev/docs/cli/graphback-cli).
+The plugin generates a `schema.json` for models annotated with `@datasync-client` in your graphql schema.
+Also, a `config.ts` file that instantiates all the models with default settings is generated.
+You can import models from `config.ts` and start coding! 
+
+### Installing the plugin
+
+For npm
+`npm install offix-datasync-client-plugin`
+or yarn
+`yarn add offix-datasync-client-plugin`
+
+Add the plugin to your `.graphqlrc.yml`
```

Comment:
you will not have .graphqlrc.yml. That is why we need small wrapper that will create it with some sensible defaults for our plugin

Reply:
This can be separate issue